tests/e2e/test_branch_self_powershell_bug.py

```
@@ -60,7 +60,7 @@ def test_branch_self_powershell_bug():
     import bots.tools.terminal_tools
 
     # Create a bot with the terminal tools (includes execute_powershell)
-    bot = AnthropicBot(model_engine="claude-haiku-4-5-20251001", max_tokens=4000, temperature=0.0)
+    bot = AnthropicBot(model_engine=Engines.CLAUDE45_HAIKU, max_tokens=4000, temperature=0.0)
```

Ruff (0.15.9)

[error] Undefined name `Engines`

(F821)